PHP/PHP Source.sublime-syntax

```diff
@@ -2576,12 +2576,14 @@ contexts:
         2: meta.regex.modifier.php
         3: punctuation.definition.string.end.php
       pop: 1
-    - match: \"
+    - match: /|"
       fail: branch-point-double-quoted
     - match: ''
       push: scope:source.regexp.php
       with_prototype:
-        - match: (?=/{{regex_modifier}}"|")
+        # for syntax test in regex
+        - match: ^// *\^.*$
```

Comment:
We should probably better refactor affected test cases instead of injecting non-regular patterns just to support existing ones.

Maybe ...

```php
<?php

$regex = '/
    a{,6}
/ux';
// <- meta.string.php string.quoted.single.php punctuation.definition.string.regex-delimiter.end.php
 // <- meta.string.php string.quoted.single.php meta.regex.modifier.php
//^ meta.string.php string.quoted.single.php meta.regex.modifier.php
// ^ meta.string.php string.quoted.single.php punctuation.definition.string.end.php

$regex = '/
    a{,6}
    b{3,} # this is comment
    c{3,6}/ux';
//   ^^^^^ keyword.operator.quantifier.regexp
```

Reply:
I think I found another bug. 

```php
$regex = '/a# this is comment/ux'; // matches 'a'
$foo = preg_replace($regex, 'b', 'aaaaaa');
echo $foo; // bbbbbb, so "# this is comment" is indeed a comment
```

```php
$regex = '/
    c{3,6} # this is comment/ux';
//   ^^^^^ keyword.operator.quantifier.regexp
//         ^^^^^^^^^^^^^^^^^ comment (at this moment this failed)
```

Reply:
> I think I found another bug.
> 
> ```
> $regex = '/a# this is comment/ux'; // matches 'a'
> $foo = preg_replace($regex, 'b', 'aaaaaa');
> echo $foo; // bbbbbb, so "# this is comment" is indeed a comment
> ```
> 
> ```
> $regex = '/
>     c{3,6}# this is comment/ux';
> //   ^^^^^ keyword.operator.quantifier.regexp
> //        ^^^^^^^^^^^^^^^^^ comment (at this moment this failed)
> ```

Resolved in 35bebe99c579d2a7ecedef041af5f8321051db20